<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sintoma CRUD</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/signo-sintoma.png</normaloff>:/icons/icons/signo-sintoma.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="symptomLabel">
       <property name="text">
        <string>Sintoma:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="symptomInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>Borrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>Crear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="modifyButton">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="symptomsTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>341</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
